packages/std/src/math/int64.rs

```diff
@@ -0,0 +1,1047 @@
+use forward_ref::{forward_ref_binop, forward_ref_op_assign};
+use schemars::JsonSchema;
+use serde::{de, ser, Deserialize, Deserializer, Serialize};
+use std::fmt;
+use std::ops::{
+    Add, AddAssign, Div, DivAssign, Mul, MulAssign, Neg, Not, Rem, RemAssign, Shl, ShlAssign, Shr,
+    ShrAssign, Sub, SubAssign,
+};
+use std::str::FromStr;
+
+use crate::errors::{DivideByZeroError, DivisionError, OverflowError, OverflowOperation, StdError};
+use crate::{forward_ref_partial_eq, Uint64};
+
+/// An implementation of i64 that is using strings for JSON encoding/decoding,
+/// such that the full i64 range can be used for clients that convert JSON numbers to floats,
+/// like JavaScript and jq.
+///
+/// # Examples
+///
+/// Use `from` to create instances of this and `i64` to get the value out:
+///
+/// ```
+/// # use cosmwasm_std::Int64;
```

Comment:
Nit: is this `#` a typo? Seen here and elsewhere.

Reply:
No, it [hides the line in the generated docs](https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#hiding-portions-of-the-example), because it is needed to compile the code, but not really relevant for the example itself.